environments/daml-sdk-env/build/1_installprerequisites.sh.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Following https://docs.azul.com/zulu/zuludocs/ZuluUserGuide/PrepareZuluPlatform/AttachAPTRepositoryUbuntuOrDebianSys.htm
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9
+sudo apt-add-repository 'deb http://repos.azulsystems.com/ubuntu stable main'
+
+# Following https://classic.yarnpkg.com/en/docs/install/#debian-stable
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```

Comment:
Is that for the new GTG or is there something else where we need this?

Reply:
Yes, this is for the GSG